modules/ossm-observability-access.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+//- ossm-observability.adoc
+
+[id="ossm-observability-access-console_{context}"]
+= Accessing the Kiali console
+
+To access the console, in the menu bar, click the *Application launcher* > *Kiali*.
+
+Select the project for your service in the *Namespace* field. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console. If you have installed the Bookinfo example, select `bookinfo`.
+
+.Procedure from the command line
+
+. Run this command from the CLI to obtain the route and Kiali URL:
++
+----
+$ oc get routes
```

Comment:
Could we add some sample output? Especially since you reference the HOST/PORT column.

Reply:
I left that out on purpose. I think that's enough context to find the link.